tsl/test/expected/compression_bgw.out

```diff
@@ -260,6 +261,7 @@ SELECT COUNT(*) AS dropped_chunks_count
                    14
 (1 row)
 
+-- Test that adding a compress chunks policy
```

Comment:
Incomplete sentence? Not sure why this is added

Reply:
Removed.